### DISCUSSION TITLE
Changing gene in genes tab should effect gene\atlas

### DISCUSSION THREAD

**Discussion by drorbar** [tanaylab/MCView#120] [Ideas]
In the Gens tabs, if a user change the selected gene in the gene\gene box or the Atlas gene\gene it should change on the second one to prevent just doing the same selection over and over again

**Comment by aviezerl** [comment]
Hmm...

Can you imagine a scenario where the user would want to show different
genes in the atlas and the query? If not - I agree, but if it is probable -
we should make it optional.

**Reply by drorbar** [reply]
We talked - seemed that we need this to be optional but the default should be on
